src/platforms/common/enriching-events/identify-user.mdx

```diff
@@ -35,13 +35,21 @@ Users consist of a few critical pieces of information that construct a unique id
 
 `ip_address`
 
-: The user's IP address. If the user is unauthenticated, Sentry uses the IP address as a unique identifier for the user. Sentry will attempt to pull this from the HTTP request data, if available. That might require <PlatformIdentifier name="send-default-pii" /> set to `true` in the SDK options. <PlatformSection notSupported={["php"]}> If set to `"{{auto}}"`, Sentry will infer the IP address from the connection between your app and Sentry's server, which is useful for client-side applications such as JavaScript applications running on Web browsers, desktop applications, or mobile apps. </PlatformSection>
+: The user's IP address. If the user is unauthenticated, Sentry uses the IP address as a unique identifier for the user. Sentry will attempt to pull this from the HTTP request data (`request.env.REMOTE_ADDR` field in JSON), if available. That might require <PlatformIdentifier name="send-default-pii" /> set to `true` in the SDK options.
 
-<PlatformSection supported={["apple"]}>
+If the user's `ip_address` is set to `"{{auto}}"`, Sentry will infer the IP address from the connection between your app and Sentry's server. For certain SDKs, particularly iOS and frontend JavaScript, this is the default behavior and requires no further configuration. For most other SDKs the default is not to collect any IP address. If you find that you're missing the user's IP address, setting this value to `"{{auto}}"` explicitly might help.
```

Comment:
this paragraph should belong under `ip_address`, but it looks like "breaks out" of that section and is de-indented:

<img width="782" alt="Screenshot 2022-01-03 at 11 55 45 AM" src="https://user-images.githubusercontent.com/5597345/147957813-d1b2c639-9945-4a3b-bb51-5b5f87b416fc.png">

i think this could cause a bit of confusion - is there a way to fix this?

Reply:
Indenting in markdown without showing anything isn't always very straightforward in Markdown, but I wonder if the `: ` would work on the following paragraphs.

Reply:
I found a component in our docs page that works like the `: `, but allows for paragraphs.